### DISCUSSION TITLE
is this bug? flatpak version not working in arch linux

### DISCUSSION THREAD

**Discussion by ghost** [tenacityteam/tenacity-legacy#726] [Q&A]
![2022-06-22-113409_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/174983869-19b75f9b-ce65-4fd3-824f-272b5963e5e1.png)

**Comment by poyribn** [comment]
i feel like i don't really see the error.
¿is it just stuck not opening Tenacity
and not writing any messages into the Terminal
after you typed the command ``org.tenacityaudio.Tenacity``?

**Reply by ghost** [reply]
yeap there is no error message just can't open
